schema.json

```diff
@@ -0,0 +1,270 @@
+{
+    "workshop": {
+	"type": "dict",
+	"required": true,
+	"schema": {
+	    "schema": {
+		"type": "dict",
+		"required": true,
+		"schema": {
+		    "version": {
+			"allowed": [
+			    "2020.02.23"
+			],
+			"required": true
+		    }
+		}
+	    }
+	}
+    },
```

Comment:
I added a `version` property inside a `schema` property inside of a new `workshop` property.  In the future we could add a `properties` field (or similar) to the `workshop` section to provide further manipulation of how workshop behaves beyond what the command line arguments do.